### ISSUE TITLE
[v86d] Is v86d actually needed for MHWD to function on ARM?

### ISSUE THREAD

**Issue by Strit** [manjaro-arm/packages-extra#6]
As the title suggests.
The v86d package does not build on arm, because of some LRMI stuff not supported on ARM.

Question is, mhwd builds fine without it (even though it's a dependency to the manjaro linux version of mhwd), so is it really needed? What I mean is, will mhwd still function on ARM the way we want it to, without this package?

Testing required.

**Comment by dodgejcr** [comment]
Well it's for some userspace x86 kernel driver stuff and since we won't be using mhwd for kernel just yet, probably don't need it. If mhwd works without it, I'm fine with that.

**Comment by Strit** [comment]
Haven't actually tested how mhwd works on Manjaro-ARM. Have you?

**Comment by dodgejcr** [comment]
Not in a while. Maybe back in march. Since we didnt have any real hardware components for it, couldnt test much. It was mainly used for Manjaro Settings Manager which was working fine (except for kernel/hardware sections)

**Comment by Strit** [comment]
It seems that mhwd does function without v86d, but it gives lots of errors/warnings because of the GPU stuff imbedded deeply into mhwd. 

The mhwd software does not even have a purpose on ARM at the moment either, so we should consider not including it in new releases until that's resolved.

As for the v86d package, I have removed it from this repository. It did not build, and was not needed anyway.

Closing.
